.github/workflows/build_and_test.yml

```diff
@@ -268,37 +266,53 @@ jobs:
     - name: Build and run tket proptests
       run: conan create --profile=tket recipes/tket-proptests --build=missing -o boost:without_fiber=True -o boost:without_json=True -o boost:without_nowide=True
     - name: Build pytket (3.8)
-      if: github.event_name == 'pull_request' || github.event_name == 'push'
+      if: github.event_name == 'push'
       run: |
         eval "$(pyenv init -)"
         pyenv shell tket-3.8
         OPENBLAS="$(brew --prefix openblas)" pip install -U scipy
         cd pytket
         pip install -e . -v
     - name: Test pytket (3.8)
-      if: github.event_name == 'pull_request' || github.event_name == 'push'
+      if: github.event_name == 'push'
       run: |
         eval "$(pyenv init -)"
         pyenv shell tket-3.8
         cd pytket/tests
         pip install -r requirements.txt
         pytest --ignore=simulator/ --doctest-modules
     - name: Build pytket (3.9)
-      if: github.event_name == 'schedule'
+      if: github.event_name == 'pull_request'
       run: |
         eval "$(pyenv init -)"
         pyenv shell tket-3.9
         OPENBLAS="$(brew --prefix openblas)" pip install -U scipy
         cd pytket
         pip install -e . -v
     - name: Test pytket (3.9)
-      if: github.event_name == 'schedule'
+      if: github.event_name == 'pull_request'
       run: |
         eval "$(pyenv init -)"
         pyenv shell tket-3.9
         cd pytket/tests
         pip install -r requirements.txt
         pytest --ignore=simulator/ --doctest-modules
+    - name: Build pytket (3.10)
+      if: github.event_name == 'schedule'
```

Comment:
the others are on "pull_request" but this is still "schedule"?

Reply:
The pattern is: 3.8 for pushes, 3.9 for PRs, and 3.10 for the weekly scheduled run.